spec/index_spec.rb

```diff
@@ -372,4 +372,51 @@
       )
     end
   end
+
+  describe Daru::CategoricalIndex do
+    before do
+      @idx = Daru::CategoricalIndex.new(
```

Comment:
In all of the code below you refer as `idx1` to what is defined here as `idx`, as far as I can tell